lib/modules/manager/mix/extract.spec.ts

```diff
@@ -25,8 +25,8 @@ describe('modules/manager/mix/extract', () => {
         {
           currentValue: '>2.1.0 or <=3.0.0',
           datasource: 'hex',
-          depName: 'ecto',
-          packageName: 'ecto',
+          depName: 'foo_bar',
+          packageName: 'foo_bar',
```

Comment:
why dep names are changed?

Reply:
I updated the dependencies in the `mix.exs` [fixture](https://github.com/renovatebot/renovate/pull/27924/files#diff-2d1a0f690965c612cc733c624ead42b46a906f2f2d889d6fad624721b8c74f8c) to be a valid. The previous version had two references to the `ecto` package, one to the Hex version and one to a Github version, which would fail to resolve in a real project.